.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more git_hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+        exclude: conda-recipe/rsmtool/meta.yaml
+    -   id: check-added-large-files
```

Comment:
Looks like the default limit here is 500 kB. Does that cover all of our test data?

Reply:
How about also adding `debug-statements`? I have done that in the past.

Reply:
For each check, I ran the check on every single file, so the `check-added-large-files` check was run on all test data. We can increase this limit, though, if you want.

I will add in `debug-statements`.

Reply:
Ah, if that was already run then it's fine. We can tweak the limit in future PRs if and when we add files larger than the default limit.